pkg/storage/unified/sql/db/trace.go

```diff
@@ -0,0 +1,42 @@
+package db
+
+import (
+	"context"
+	"database/sql"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+const tracePrefix = "stdlib.database.sql.db."
+
+type traceDB struct {
```

Comment:
TODO: we may want to add an unexported method like `getSpan` that calls `b.tracer.Start` and adds the database driver name and (somehow) the database version as well. Down the line, this can be useful in debugging and to compare behaviour after databases upgrades.